react_practice/react-apps/waitlist-page-v1/src/PrimaryBtn.js

```diff
@@ -1,11 +1,12 @@
 import "./PrimaryBtn.css";
+import cx from "classnames";
 
-function PrimaryButton({ title, handleSubmit }) {
+function PrimaryBtn({ className, title, onClick }) {
   return (
-    <div className="primary-btn-header" handleSubmit={handleSubmit}>
-      <h5 className="primary-btn-title">{title}</h5>
-    </div>
+    <button className={cx("primary-btn", className)} onClick={onClick}>
+      {title}
+    </button>
```

Comment:
This is the crux of where you were seeing your problems and why you were having a difficult time getting the click behavior to work correctly. Let me start with the event handling.

I see that you were passing the `handleSubmit` handler to a prop on the `div` called `handleSubmit`. The reason this did not work is because the `div` does not have a prop called `handleSubmit` that does anything. In order for this to work, you would have wanted to put this on the `onClick` prop of the `div`

```js
// onClick is something that `div` understands
<div onClick={handleClick}>
```

If you're curious about the full list of events supported by the primitive React components (any lowercase name like `div`, `button`, `span`, etc.), check out [these docs](https://reactjs.org/docs/events.html#supported-events)!

You'll also note here that I switched up the elements used for this button. I am no longer rendering a `div` and an `h5`, but rather a plain `button` element. This is more semantically meaningful. You generally do not want to render headings inside a button as they should be reserved for... well headings. My guess is you were using it as a means to control the `font-size`, but it would be best to put that `font-size` on the button styling itself.